Sources/PointFree/Admin/Index.swift

```diff
@@ -49,7 +50,7 @@ let showNewEpisodeEmailMiddleware =
 
 private let showNewEpisodeView = View<Database.User> { currentUser in
   ul(
-    episodes
+    AppEnvironment.current.episodes()
```

Comment:
I hide the episodes inside the environment so that we can control it in tests

Reply:
Any reason why it's a function and not a property?

Reply:
just anticipating this function taking arguments some day for filtering by tags / sorting by fields etc...